### DISCUSSION TITLE
通过购买任意金额的兑换码，使用订单编号和兑换码不能登录

### DISCUSSION THREAD

**Discussion by Lion066** [BurnHair/docs#101] [技术支持]
我之前是用GitHub授权登录的，后来购买了兑换码，兑换以后，尝试使用订单编号和兑换码登录，发现不能登录，提示「错误：用户名或密码错误，或用户已被封禁」，重新使用原来的GitHub授权登录又没问题。

**Comment by BennyThink** [comment]
兑换了以后肯定不能再用兑换码登录了呀🤣 因为用兑换码登录，兑换码的余额会直接加给这个账户的，已经兑换的就失效了
